timeHelper.go

```diff
@@ -3,7 +3,8 @@ package gocron
 import "time"
 
 type timeHelper interface {
-	Now() time.Time
+	Now(*time.Location) time.Time
+	NowRoundedDownToSeconds(*time.Location) time.Time
```

Comment:
another thought: as we don't consider nanoseconds anywhere in the project, what do you think about putting this logic in the Now() method and removing this one? Seems like a good idea?

Reply:
Yeah I was thinking about that last night also. Glad you are validating it. I'll make that change as well.